workflows/hifiasm/triobev.wdl

```diff
@@ -0,0 +1,133 @@
+version 1.0
+
+#This new workflow is used to run yak_triobin and yak_trioeval for memory size testing, coded by Charlie Bi
+
+task yak_trioeval {
```

Comment:
Is this just for testinging @cbi-star ?

Reply:
> Overall looks good. Had some questions and I think you can simplofy the imports since they are all relative to the wdl you are refereing to.

I will be working on this. Thank you, Venkat!

Reply:
Yes. The workflows yak_trioeval and yak_triobin are for testing only. We increase the memory size and the memory problem solved.